extensions/applicationinsights-cfgsync-js/src/CfgSyncPlugin.ts

```diff
@@ -75,6 +76,14 @@ export class CfgSyncPlugin extends BaseTelemetryPlugin implements ICfgSyncPlugin
                 return _mainConfig;
             }
 
+            _self.pause = () => {
+                _paused = true;
+            }
+
+            _self.resume = () => {
+                _paused = false;
```

Comment:
These probably should also stop / resume any timer for the CDN lookup (if configured that way)